setup.py

```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup
+
+setup(
+    name='nerdouts',
+    version='0.1dev',
+    packages=['nerdouts'],
+    license='Creative Commons Attribution-Noncommercial-Share Alike license',
+    scripts=['nerdouts/workout'],
+    data_files=[('.', ['README.md'])],
+    long_description=open('README.md').read(),
```

Comment:
Neat trick.  I usually put the documentation at the top of the file use `__doc__`

Reply:
oh interesting. Actually think I might prefer that